lib/puppet/provider/parsedfile.rb

```diff
@@ -21,11 +21,9 @@ class Puppet::Provider::ParsedFile < Puppet::Provider
   extend Puppet::Util::FileParsing
 
   class << self
-    attr_accessor :default_target, :target, :raise_prefetch_errors
+    attr_accessor :default_target, :raise_prefetch_errors
   end
 
-  attr_accessor :property_hash
```

Comment:
Not sure why it needs to be an accessor. I think it can only break things.

Reply:
I agree in principal, but it's referenced by subclasses of ParsedFile like https://github.com/puppetlabs/puppetlabs-sshkeys_core/blob/fbd0c6dc9ca7314a567b6873cb66e2adf52af30b/lib/puppet/provider/sshkey/parsed.rb#L32C8-L32C21 and is therefore part of the parsed file API.